test/realtime/server_test.exs

```diff
@@ -22,17 +79,16 @@ defmodule Realtime.ServerTest do
 
       bypass = Bypass.open()
       url = "http://localhost:#{bypass.port}/VehiclePositions.json"
+      Application.put_env(:skate, "CONCENTRATE_VEHICLE_POSITIONS_URL", url)
 
-      Bypass.expect(bypass, fn conn ->
+      Bypass.stub(bypass, "GET", url, fn conn ->
```

Comment:
What's the difference between `expect` and `stub`? The [docs](https://hexdocs.pm/bypass/Bypass.html#stub/4) are missing for both functions.

Reply:
There is some better documentation on the GitHub project: https://github.com/PSPDFKit-labs/bypass#stub4-bypass_instance-method-path-function

Basically `expect` stubs the request and also sets a test expectation. `stub` just stubs, it doesn't care if it gets called or not.

In this case we want to stub this request because the concentrate system will ping it in the background, but it isn't a necessary condition for our test (and depending on how long our test takes to run it might or might not ever actually have a chance to get called.